src/i18n/locales/nl.json

```diff
@@ -136,7 +136,7 @@
   "settings.service.form.enableService": "Activeer service",
   "settings.service.form.enableNotification": "Activeer meldingen",
   "settings.service.form.team": "Team",
-  "settings.service.form.customUrl": "Custom server",
+  "settings.service.form.customUrl": "Aangepaste server",
```

Comment:
My dutch skills are non existent but I'm not sure if 'Aangepaste' would be the proper translation as this label should reflect the meaning of:
- I'm having my own server
- I'm using an on-premise server

Reply:
Technically "custom" is not a word that exists in Dutch. So it should have been translated to something else.
However, as @adlk states correctly, "custom" means more than just "aangepast", and in this context I would prefer something like "Eigen/lokale server" as alternative string. More like literally the meaning you state, Stefan. 
"Aangepast" would be translated back to "Changed", and that is not correct in this context.